hack/go-lint.sh

```diff
@@ -13,6 +13,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+which golint > /dev/null || GO111MODULE=off go get -u golang.org/x/lint/golint
```

Comment:
sorry nitpick, can we have `command -v` instead like we have for other commands?

Reply:
All our scripts are using `which`. I kept for cohesion. I know that the shell checker complains about this, I will change everything.

Reply:
Ah yes, sorry I was thinking about nexus-operator :p